### PR TITLE
Usage data/backend insert

### PR DESCRIPTION
**Pull request by coury-clark**
Adds default values to the `InsertID` and `DeviceID` similar to how the webapp behaves for backend generated events. This is required such that downstream consumers can deduplicate these events.

## Test plan

Started Sourcegraph and ensured all backend events were inserting with ids

image

### CODE REVIEW

File: client/web/src/user/settings/backend.tsx [hunk 1]
```diff
@@ -200,8 +200,8 @@ function createEvent(event: string, eventProperties?: unknown, publicArgument?:
         source: EventSource.WEB,
         argument: eventProperties ? JSON.stringify(eventProperties) : null,
         publicArgument: publicArgument ? JSON.stringify(publicArgument) : null,
-        deviceID: window.context.sourcegraphDotComMode ? eventLogger.getDeviceID() : null,
-        eventID: window.context.sourcegraphDotComMode ? eventLogger.getEventID() : null,
-        insertID: window.context.sourcegraphDotComMode ? eventLogger.getInsertID() : null,
+        deviceID: eventLogger.getDeviceID(),
```
Comment (chwarwick) [inline client/web/src/user/settings/backend.tsx:203]: Is there a reason this was specifically not done outside of sourcegraph.com?

File: client/web/src/user/settings/backend.tsx [hunk 2]
```diff
@@ -200,8 +200,8 @@ function createEvent(event: string, eventProperties?: unknown, publicArgument?:
...
+        eventID: eventLogger.getEventID(),
```
Comment (chwarwick) [inline client/web/src/user/settings/backend.tsx:204]: I didn't see this get used again eventID sounds like a field that would typically get saved with the event.

**Review by chwarwick — APPROVED** [review]

File: client/web/src/user/settings/backend.tsx (hunk 2 above)
Comment (coury-clark) [reply client/web/src/user/settings/backend.tsx:204]: It isn't used AFIAK. I have no context on why or what this is for, and am going to leave it for the data team to figure out and solve.

File: client/web/src/user/settings/backend.tsx (hunk 1 above)
Comment (coury-clark) [reply client/web/src/user/settings/backend.tsx:203]: No, the API is capable of accepting this input in both environments.

**Pull request merged**